packages/fxa-content-server/app/scripts/views/mixins/coppa-mixin.js

```diff
@@ -121,6 +121,8 @@ export default function(config = {}) {
           keyCode === KeyCodes.TAB ||
           keyCode === KeyCodes.LEFT_ARROW ||
           keyCode === KeyCodes.RIGHT_ARROW ||
+          keyCode === KeyCodes.UP_ARROW ||
+          keyCode === KeyCodes.DOWN_ARROW ||
```

Comment:
missing these caused the coppa to be a black hole for the remote control

Reply:
The best part about this change is that up/down arrow on Fx Desktop now cause the value to increment/decrement as you'd expect in a number input! 💯